cql3/statements/select_statement.hh

```diff
@@ -47,6 +47,7 @@
 #include <seastar/core/shared_ptr.hh>
 #include "transport/messages/result_message.hh"
 #include "index/secondary_index_manager.hh"
+#include "service/forward_service.hh"
```

Comment:
Please use forward declarations (!) if possible.

Reply:
This include was dropped, because `parallelized_select_statement` (which used `forward_service`) was moved to `.cc`.